clang/include/clang/Basic/Attr.td

```diff
@@ -1654,6 +1654,57 @@ def SYCLIntelFPGAMaxConcurrency : Attr {
   let Documentation = [SYCLIntelFPGAMaxConcurrencyAttrDocs];
 }
 
+def SYCLIntelFPGALoopCoalesce : Attr {
+  let Spellings = [CXX11<"intelfpga","loop_coalesce">];
+  let Args = [ExprArgument<"NExpr">];
+  let LangOpts = [SYCLIsDevice, SYCLIsHost];
+  let HasCustomTypeTransform = 1;
+  let AdditionalMembers = [{
+    static const char *getName() {
```

Comment:
These methods are no more required, i suppose.

Reply:
No, these methods are used in `CheckForDuplicationSYCLLoopAttribute` function in `SemaStmtAttr.cpp`

Reply:
Hm, why is this function not using getSpelling instead? But this is for future improvement then.